spec/services/zendesk.rb

```diff
@@ -12,10 +12,8 @@
     def mock_zendesk_response(body)
       stub_request(
         :get,
-        /.*zendesk.com\/api\/v2\/.*/
-      ).with(
-        :body => "{\"query\":\"query:created>7days group:\\\"Incident response and tuning\\\" type:ticket\"}"
-      ).to_return(:status => 200, :body => body, :headers => {"Content-Type" => "application/json"})
+        /.*zendesk.com\/api\/.*/
```

Comment:
It is executed when I run `rspec`.

I think I excluded it in the stub in case the version number changes, but it's not very likely.